src/sql/workbench/parts/query/common/resultSerializer.ts

```diff
@@ -212,7 +212,7 @@ export class ResultSerializer {
 		} else if (format === SaveFormat.XML) {
 			saveResultsParams = this.getConfigForXml();
 		}
-		return saveResultsParams;
+		return saveResultsParams!; // this could be unsafe
```

Comment:
>// this could be unsafe [](start = 29, length = 23)

Feels like we should log an error at the very least or something if it's an unknown save format